src/test/ui/error-codes/E0225.rs

```diff
@@ -8,7 +8,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#![feature(trait_alias)]
+
+trait Foo = std::io::Read + std::io::Write;
+trait Bar = Foo;
+
 fn main() {
     let _: Box<std::io::Read + std::io::Write>;
-    //~^ ERROR only auto traits can be used as additional traits in a trait object [E0225]
+    let _: Box<Bar>;
```

Comment:
Please add a `//~ ERROR E0225` on this line

Reply:
Done.